extras-requirements.txt

```diff
@@ -0,0 +1 @@
+pollination-honeybee-vtk==0.4.5
```

Comment:
Don't we need this in any case? Is there a case where you don't want to generate the results visualization part?